content/doc/developer/plugin-development/pipeline-integration.adoc

```diff
@@ -2,12 +2,579 @@
 layout: developerguide
 title: Writing Pipeline-Compatible Plugins
 references:
-- url: https://github.com/jenkinsci/pipeline-plugin/blob/master/DEVGUIDE.md
-  title: Plugin Developer Guide
 - url: https://github.com/jenkinsci/workflow-step-api-plugin/blob/master/README.md
   title: Writing Pipeline steps
+- url: https://github.com/jenkinsci/workflow-basic-steps-plugin/blob/master/CORE-STEPS.md
+  title: Core Pipeline steps
+- url: http://javadoc.jenkins-ci.org/jenkins/tasks/SimpleBuildStep.html
+  title: SimpleBuildStep javadoc
+- url: https://github.com/jenkinsci/workflow-scm-step-plugin/blob/master/README.md
+  title: SCM steps and Pipeline
```

Comment:
Again sort of tangential relevance here; maybe best kept inline in the SCM section.